legal-api/report-templates/amalgamationApplication.html

```diff
@@ -1,13 +1,241 @@
+                          <div>
+                            {{ party.deliveryAddress.addressCity }}
+                            {{ party.deliveryAddress.addressRegion }}
+                            &nbsp;{{ party.deliveryAddress.postalCode }}
+                          </div>
+                          <div>{{ party.deliveryAddress.addressCountry }}</div>
+                          <div class="delivery-instructions pt-1">{{ party.deliveryAddress.deliveryInstructions }}</div>
+                        </td>
+                      {% endif %}
+                      </tr>
+                </table>
+            {% endif %}
+         {% endfor %}
+      {% endfor %}
+              </div>
+
+   <!-- CompletingParty Statement [[amalgamation/completingPartyStmt.html]] -->
+    <div class="no-page-break">
+      <div class="separator mt-4"></div>
+        <div class="section-title mt-4">
+          <span>Completing Party Statement</span>
+          <div class="section-data mt-3">
+            I,{% for party in amalgamationApplication.parties %} 
+                {% for party_role in party.roles %}
+                  {% if party_role.roleType == 'Completing Party' %}
+                    <span class="capitalize-text">{{ party.officer.lastName }}</span>,
+                    <span class="capitalize-text">{{ party.officer.firstName }}</span>
+                    {% if party.officer.middleName %}
+                      <span class="capitalize-text">{{ party.officer.middleName }}</span>
+                    {% endif %}
+                  {% endif %}
+                {% endfor %}
+              {% endfor %}, the completing party, have examined the articles and the incorporation agreement applicable to the
+              company that is to be incorporated by the filing of the Incorporation Application and confirm that:
+              <table class="certify-statement-table" role="presentation">
+                 <tr>
+                    <td class="pt-2">a)</td>
+                    <td class="pt-2">the Articles and the Incorporation Agreement both contain a signature line for each person
+                       identified as an incorporator in the Incorporation Application with the name of that person set out
+                       legibly under the signature lines,</td>
+                    </tr>
+                 <tr>
+                    <td class="pt-2">b)</td>
+                    <td class="pt-2">an original signature has been placed on each of those signature lines, and</td>
+                 </tr>
+                 <tr>
+                    <td class="pt-2">c)</td>
+                    <td class="pt-2">I have no reason to believe that the signature placed on a signature line is not the
+                       signature of the person whose name is set out under that signature line.</td>
+                    </tr>
+              </table>
+            </div>
+          </div>
+
+  </body>
 </html>
```

Comment:
ALso had the NOA here, but must confirm with Yui if they want in a part of the Amalgamation Application as in the UXPin, or in a separate document first.

Reply:
Confirmed with Yui that we want NOA at the end of the application as well as in its own separate file.